src/modules/rewards/Rewards.constants.ts

```diff
@@ -26,3 +27,27 @@ export const ActivityTypeID = {
     image: Twitter
   }
 };
+
+interface Illustration {
+  JoinURL: string;
+  name: string;
+}
+
+interface Illustrations {
+  [key: string]: Illustration;
+}
```

Comment:
uses type instead of interface @abhishek-01k. Also I already added a comment for this we need to replace the activity with the activityType